### DISCUSSION TITLE
Autotracking different zones for different objects

### DISCUSSION THREAD

**Discussion by jakeycrx** [blakeblackshear/frigate#13360] [General]
Hi,

Is it possible to set seperate zones for autotracking different objects like you can with the standard zones and objects?

I would like to autotrack cats in the cameras full field of view, but would only like to autotrack a person in a smaller area (my driveway)

**Comment by NickM-27** [comment]
yes, you can assign objects to each zone, so just make both of those zones work for autotracking

**Comment by hawkeye217** [comment]
Just make sure you set both of those zones in the `required_zones` in your autotracking config.

**Comment by jakeycrx** [comment]
So set the objects and zones in the standard setup? I guess I leave the objects blank in the autotracking section?

One other thing, is there any way to deal with zones moving with the camera when it moves? Or is it just something we have to accept when autotracking?

**Reply by hawkeye217** [reply]
> So set the objects and zones in the standard setup? I guess I leave the objects blank in the autotracking section?

You should fill out `objects` in the `autotracking:` section to be all of the objects you want to autotrack, no matter the zone it enters. 

Zones on PTZ cameras are usually not applicable - precisely because zones assume a fixed position. Autotracking uses zones only as a trigger. The PTZ starts is in its "home" position and frigate assumes it's a static camera until it begins moving. Autotracking doesn't use zones once it's actively tracking an object.

On my PTZ with autotracking, I have one zone (my lawn) and one motion mask (my camera's timestamp).

**Reply by jakeycrx** [reply]
All sorted Thanks!
